MNTS-90869 Link to Custom Schedules (#24542)
DataDog/documentation: content/en/monitors/guide/template-variable-evaluation.md

{{/is_match}}
```

> **Note:** If you are looking for a way to evaluate your monitor on a schedule, you may want to look at [Custom Schedules][2] instead

Reviewer: ```suggestion
**Note:** If you need to evaluate your monitor on a schedule, see [Custom Schedules][2] instead.
```
